frontend/common/src/components/SideMenu/sideMenu.css

```diff
@@ -1,98 +1,130 @@
-.side-menu {
-  bottom: 0;
-  flex-shrink: 0;
+/* [data-h2] .side-menu {
+    bottom: 0;
+    flex-shrink: 0;
+    height: 100vh;
+    max-width: 95vw;
+    overflow-x: hidden;
+    position: fixed;
+    left: 0;
+    top: 0;
+    transform: translateX(-100%);
+    transition: transform 150ms ease-in;
+    width: 16rem;
+    z-index: 9999;
+} */
+
+[data-h2] .side-menu {
```

Comment:
Why  are [data-h2] selectors needed for files like this with no relation to hydrogen? Can we just leave them out to reduce noise and special-looking-ness of our styling?

Reply:
I've since cleaned that CSS file up - though `[data-h2]` is and will be necessary when you have to overwrite styles that are set by Hydrogen attributes.